moveit_core/trajectory_processing/include/moveit/trajectory_processing/iterative_time_parameterization.h

```diff
@@ -64,4 +64,6 @@ class IterativeParabolicTimeParameterization
   double findT1(const double d1, const double d2, double t1, const double t2, const double a_max) const;
   double findT2(const double d1, const double d2, const double t1, double t2, const double a_max) const;
 };
+
+void updateTrajectory(robot_trajectory::RobotTrajectory& rob_trajectory, const std::vector<double>& time_diff);
```

Comment:
could you please turn this method into a static method inside the `IterativeParabolicTimeParameterization` class above?
Sorry for these further incremental requests. I looked at the details again to maybe find the cause for the CI failures.